api/shell.go

```diff
@@ -6,6 +6,7 @@ package api
 
 import (
 	"bytes"
+	stdContext "context"
```

Comment:
I'd suggest leaving this as context and using `tsurucontext` or similar for the tsuru one. Most developers will imediatelly think of context when they read `context.Something`.

Reply:
I just noticed the `stdcontext` approach it is used in most places that import both contexts. In this case consistency should be the priority.